zkevm-circuits/src/util.rs

```diff
@@ -392,3 +392,84 @@ pub(crate) fn unusable_rows<F: Field, C: Circuit<F>>() -> usize {
 
     cs.blinding_factors() + 1
 }
+
+/// The function of this algorithm： Split a vec into two subsets such that
+/// the sums of the two subsets are as close as possible。
+pub(crate) fn find_two_closest_subset(vec: &[usize]) -> (Vec<usize>, Vec<usize>) {
```

Comment:
did we consider using https://en.wikipedia.org/wiki/Greedy_number_partitioning instead? is it possible for total_sum / vec.len() to be so large that this function takes too much time/memory?

Reply:
usually just at most dozens of bytecodes.   Very small

Reply:
agree with you. current time complexity is related to sum.  too much..   This function should be run in CCC, need to be fast

@DreamWuGit can find a faster algorithm

Reply:
use simple algorithm in  https://github.com/scroll-tech/zkevm-circuits/pull/1368/commits/5fefdcd4f4483e07b2b6a9cfac2291aa3516069b